server/pulp/server/db/querysets.py

```diff
@@ -116,6 +119,27 @@ def get_or_404(self, **kwargs):
             raise pulp_exceptions.MissingResource(**kwargs)
 
 
+class WorkerQuerySet(CriteriaQuerySet):
+    """
+    Custom queryset for workers
+    """
+
+    def get_workers_online(self):
+        """
+        Returns a queryset that contains only worker documents that have been updated in the last
+        25 seconds.
+
+        :return: mongoengine queryset object
+        :rtype:  mongoengine.queryset.QuerySet
+        """
+        query_set = self
```

Comment:
This read strangely to me. I think having L#138 be `self.filter....` would be fine. This is not a required change.